src/robusta/core/playbooks/playbooks_event_handler_impl.py

```diff
@@ -168,7 +170,13 @@ def __run_playbook_actions(
                     execution_event.response = self.__error_resp(msg)
                     continue
 
-                registered_action.func(execution_event, params)
+                try:
+                    registered_action.func(execution_event, params)
+                except Exception:
+                    logging.error(f"Failed to execute action {action.action_name} {action_params}", exc_info=True)
+                    execution_event.add_enrichment(
+                        [MarkdownBlock(text=f"Oops.. Error processing {action.action_name}")]
```

Comment:
English nitpick: should be `...` and not `..` IMO

Reply:
Can we add the exception here as a FileBlock? It's the first thing I would want to see if this happened.

Reply:
regarding the ..., sure
I don't think this is the place to add the exception.
This is good for the Robusta developer, but for most users, they don't care about that exception. 
Most of the times it's an SRE or a developer, who doesn't care about Robusta.
It will only confuse them IMO. 
WDYT?

Reply:
I think we should still add it in some way. It's the first thing we're going to need when this occurs.

What if we make it a FileBlock so it doesn't show up in the message by default? (We could give it an extension like .log and not .txt if it will prevent slack from showing a preview.) In other words, we can try to hide it from being displayed prominently, but I think it should be there.